ozone_concentrations/create_artifact.jl

```diff
@@ -7,16 +7,20 @@ using ClimaArtifactsHelper
 FILE_URLs = [
     "https://caltech.box.com/shared/static/hj2yucye0u3l8e4x8o1d0w35ijj5amh6.nc",
     "https://caltech.box.com/shared/static/h3c8rlpctxpq3uh1r01zhips426lgi0e.nc",
+    "https://caltech.box.com/shared/static/d72v4i6gy9m5ygel9tn95gjv74tpchx7.nc",
```

Comment:
We should update these links to not use the Caltech box. Otherwise, it defies the purpose of reproduciability.